src/test/java/pages/HomePage.java

```diff
@@ -0,0 +1,173 @@
+package pages;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.Keys;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+
+public class HomePage extends BasePage{
+
+    private static Boolean playlistExists = Boolean.FALSE;
+
+    //WebElements
+    @FindBy (css = "#playlists")
+    protected WebElement playlists;
+    @FindBy (css = "nav.menu.playlist-item-menu ul")
+    protected WebElement plsContextMenu;
+    @FindBy (css = "li.playlist.playlist.editing input")
+    protected WebElement newPlaylist;
+    @FindBy (css = "div.success.show")
+    protected WebElement notificationMessage;
+    @FindBy (css = "div.success.show:nth-child(2)")
+    protected WebElement notificationMessageChild2;
+    @FindBy (css = "#homeWrapper")
+    protected WebElement songsOnHomePage;
+    @FindBy (css = "#songsWrapper")
+    protected WebElement allSongs;
+    @FindBy (css = "#playlistWrapper")
+    protected WebElement playlistSongs;
+    @FindBy (css = "img[alt = 'Sound bars']")
+    protected WebElement soundBar;
+    @FindBy (css = "li[data-testid *= 'playlist-context-menu-edit']")
+    protected WebElement editPlaylist;
+    @FindBy (css = "#sidebar a.songs")
+    protected WebElement chooseAllSongs;
+    @FindBy (css = "i[title='Create a new playlist']")
+    protected WebElement createPlstButton;
+    @FindBy (css = "li[data-testid='playlist-context-menu-create-simple']")
+    protected WebElement createNewPlst;
+    @FindBy (css = "#playlistWrapper button.del.btn-delete-playlist")
+    protected WebElement deletePlstButton;
+    @FindBy (css = "span.play>i")
+    protected WebElement playSongButton;
+    @FindBy (css = "#songResultsWrapper  tr > td.title")
+    protected WebElement firstSongInSearch;
+    @FindBy (css = "#songsWrapper tr > td.title")
+    protected WebElement firstSong;
+    @FindBy (css = "button.btn-add-to")
+    protected WebElement addToPlstButton;
+    @FindBy (css = "div.alertify button.ok")
+    protected WebElement alertButtonOK;
```

Comment:
If a variable is used **only by other methods of that class**, it should be declared as **private**.